test/integration/warm-pool/test_fns.go

```diff
@@ -0,0 +1,68 @@
+package warm_pool
+
+import (
+	"fmt"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework"
+	"github.com/aws/amazon-vpc-cni-k8s/test/framework/resources/k8s/manifest"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"strconv"
+)
+
+func QuickScale(f *framework.Framework, deployBuilder *manifest.DeploymentBuilder) {
```

Comment:
nit: These functions should all return an error and the deployment that they created